webpack.config.js

```diff
@@ -0,0 +1,85 @@
+const path = require('path');
+const webpack = require('webpack');
+
+const MiniCssExtractPlugin = require("mini-css-extract-plugin");
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+module.exports = {
+  mode: 'development',
+  entry: {
+    index: './src/index.js',
+    tailwind: './src/tailwind.js'
```

Comment:
Does the sencond entry make sense? It just loads `tailwind.css` and you could do that in `index.js` directly by calling `import './css/tailwind.css';`